ocf/codec.go

```diff
@@ -24,7 +24,12 @@ const (
 	ZStandard CodecName = "zstandard"
 )
 
-func resolveCodec(name CodecName, lvl int) (Codec, error) {
+type zstdOptions struct {
+	EOptions []zstd.EOption
+	DOptions []zstd.DOption
+}
+
+func resolveCodec(name CodecName, lvl int, zstdOpts zstdOptions) (Codec, error) {
```

Comment:
Perhaps it would be best to have a `codecOptions` or `codecConfig` struct here containing the compression level and zstd options. Would make this cleaner in the long run.